robottelo/datafactory.py

```diff
@@ -70,24 +70,17 @@ def generate_strings_list(length=None, exclude_types=None, bug_id=None,
 @filtered_datapoint
 def invalid_emails_list():
     """Returns a list of invalid emails."""
-    data = [
+    return [
         u'foreman@',
         u'@foreman',
         u'@',
         u'Abc.example.com',
         u'A@b@c@example.com',
-        u'email@example..c',
         # total length 255:
         u'{0}@example.com'.format(gen_string('alpha', 243)),
         u'{0}@example.com'.format(gen_string('html')),
         u's p a c e s@example.com',
-        u'dot..dot@example.com'
     ]
```

Comment:
we can add a comment saying "emails with successive dots was removed as per bug: 1455501 CLOSED NOTABUG"